spec/acceptance/default_parameters_jira_10_spec.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+def on_supported_os
+  (fact('os.family') == 'Debian' and fact('os.release.major') == '22.04') or (fact('os.family') == 'RedHat' and fact('os.release.major') >= '8')
+end
+
+prepare = <<-EOS
+  package {'diffutils':
+    ensure  => installed
+  }
+  file_line{'enable show_diff':
```

Comment:
was that actually required? I thought beaker uses that by default

Reply:
Yes - it is for testing purposes and to see more details. I've commented it out.